src/sentry/static/sentry/app/views/health/list/commonLayout.tsx

```diff
@@ -0,0 +1,48 @@
+import styled from '@emotion/styled';
+
+import space from 'app/styles/space';
+
+const StyledLayout = styled('div')`
+  display: grid;
+  grid-template-columns: 24px 3fr 1.7fr 2fr 1.2fr 2fr 1fr 1fr 1fr;
+  grid-column-gap: ${space(1.5)};
+  width: 100%;
+  align-items: center;
+  @media (max-width: ${p => p.theme.breakpoints[2]}) {
+    grid-template-columns: 24px 3fr 1fr 1fr 1fr 1fr 1fr;
+  }
+  @media (max-width: 600px) {
```

Comment:
Can we use a breakpoint from `theme`? If not can we make it a constant local to this file?